api/client.go

```diff
@@ -299,12 +299,12 @@ type ErrorResponse struct {
 }
 
 func (r *ErrorResponse) Error() string {
-	s := fmt.Sprintf("%v %v: %d",
+	s := fmt.Sprintf("%v %v: %s",
 		r.Response.Request.Method, r.Response.Request.URL,
-		r.Response.StatusCode)
+		r.Response.Status)
 
 	if r.Message != "" {
-		s = fmt.Sprintf("%s %v", s, r.Message)
+		s = fmt.Sprintf(": %s %v", s, r.Message)
```

Comment:
Is that right? if `r.Message != ""` then `s` looks like `: <method> <url>: Bad Request <message>`

Reply:
oh shoot, no, you're right, it should be 
```suggestion
		s = fmt.Sprintf("%s: %v", s, r.Message)
```

Reply:
i had read it as concatenating, it's actually sprintfing, shows me for not actually reading the code i'm changing